osquery/dispatcher/scheduler.cpp

```diff
@@ -228,6 +234,22 @@ void SchedulerRunner::maybeFlushLogs(uint64_t time_step) {
   }
 }
 
+bool isDenylisted(const ScheduledQuery& query) {
+  if (!query.denylisted) {
+    return false;
+  }
+  if (!FLAGS_schedule_allow_denylisted_events) {
+    return true;
+  }
+
+  auto tables = QueryPlanner(query.query).tables();
+  bool event_based =
+      std::any_of(tables.begin(), tables.end(), [](const std::string& table) {
+        return boost::algorithm::ends_with(table, "_events");
```

Comment:
This should be `select name from osquery_events where type = 'subscriber';` right? Is there a simple way to get that?

Reply:
Correct, let me see if I can create a helper function for that.